andela/ah-legion-frontend
# Ft update delete 165556568

### How to manually test this:
- Clone the repository `https://github.com/andela/ah-legion-frontend`
- Checkout the branch `ft-update-delete-165556568`
- Run `npm install` to install dependecies
-  Run `npm start' to fire up the dev server
-  Go to a browser, login and visit the  url  `/articles/drafts` to see your articles. Click on the delete icon to delete an article. To edit, click on the title and you be directed to a link where you can edit the article.


### PT
[#165556568](https://www.pivotaltracker.com/story/show/165556568)
